sample/kotlin/build.gradle.kts

```diff
@@ -143,11 +143,16 @@ android {
     }
 }
 
+repositories {
+    mavenLocal()
+}
+
```

Comment:
shouldn't be needed, we consume from the project, not from the published artifact

Reply:
yes was only for local testing